README.md

```diff
@@ -121,3 +134,23 @@ its configuration and operation.
   [`tokio-console`](https://github.com/tokio-rs/console). See the
   [documentation](docs/CONFIGURING_TOKIO_CONSOLE.md) for configuration
   instructions.
+
+## Versioning and Stability
+
+Janus follows [semantic versioning](https://semver.org/). Because we are at major
+version 0, we increment the minor version number for breaking changes, and
+increment the patch version for new features and backwards-compatible bug fixes.
+
+What is considered a breaking change depends on the crate. The following crates
+are stable on their Rust API:
+- `janus_collector`
+- `janus_client`
+- `janus_messages`
+- `janus_core`
```

Comment:
Note that I've defined `janus_core` and `janus_messages` as stable. I'm not sure that we actually want this though. I would ideally like to say something like
> `janus_core` and `janus_messages` are unstable and should not be directly depended on. If you find yourself needing to depend on these packages in the course of using `janus_client` and `janus_collector`, open a bug report.

i.e. we should reexport types required by `janus_client` and `janus_collector`.

Reply:
I think we should say that `janus_messages` is stable: we originally split the `janus_messages` crate out so that one of our partners could use it in their own software, and AFAIK they are still doing so.

I'm agnostic as to whether `janus_core` should be stable -- I do not remember why we decided to publish it to crates.io. Is that information recorded anywhere?

Reply:
`janus_client` and `janus_collector` depend on `janus_core`, so it has to be on crates.io.